schema/crdb/dbinit.sql

```diff
@@ -2424,6 +2425,14 @@ CREATE INDEX IF NOT EXISTS lookup_console_by_silo_user ON omicron.public.console
     silo_user_id
 );
 
+-- We added a UUID as the primary key, but we need the token to keep acting like it did before.
+-- "When you change a primary key with ALTER PRIMARY KEY, the old primary key index becomes a secondary index."
+-- We chose to use DROP CONSTRAINT and ADD CONSTRAINT instead and manually create the index.
+-- https://www.cockroachlabs.com/docs/v22.1/primary-key#changing-primary-key-columns
+CREATE UNIQUE INDEX IF NOT EXISTS console_session_token_unique ON omicron.public.console_session (
+    token
+);
```

Comment:
turbo nit, sorry: in Rust code, we wrap lines at the 80th column. In `.sql` files, there's no formatting tool enforcing that, but it would be nice to wrap at the same column as in the Rust code for consistency, IMO.